docs/locale/zh_CN/SUT.md

```diff
@@ -0,0 +1,57 @@
+# 目前只支持 Hyperledger Fabric
+当前样本从 fabric-sample 中的 test-network-k8s 学习
+
+＃＃ 用法
+启动网络，创建通道，并部署 [basic-asset-transfer](./asset-transfer-basic) 智能合约：
+```外壳
```

Comment:
don't translate code block

Reply:
Here I don't need to translate from line no. 6-11?

Reply:
yes... https://www.markdownguide.org/getting-started/

Reply:
they are markdown formatting, we hope keep same for code block. so that user can copy paste the code block into their console/terminal for usage.